osquery/tables/networking/darwin/wifi.mm

```diff
@@ -66,6 +68,23 @@ Status getKnownNetworksKey(std::string& key) {
                                                        : INTEGER(0);
   } else if (CFGetTypeID(property) == CFStringGetTypeID()) {
     return stringFromCFString((CFStringRef)property);
+  } else if (CFGetTypeID(property) == CFArrayGetTypeID() && key == "bssids") {
+    std::string results;
+    // Fetch the first BSSID we can find
+    auto count = CFArrayGetCount((CFArrayRef)property);
```

Comment:
Does this and the one below need to be implicit casts?

Reply:
Yeah I does unfortunately.